app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/Event.kt

```diff
@@ -34,6 +35,18 @@ data class Event(@PrimaryKey
         const val defaultDuration = 7200000 // Milliseconds
     }
 
+    fun getFormattedDateTime(): String {
+        return "${getFormattedDate()}, ${getFormattedTime()}"
```

Comment:
If we use the new date formatting as suggested in my comment, I think we should remove the comma here

Reply:
Agree